src/components/select/select.js

```diff
@@ -774,7 +774,7 @@ function SelectProvider($$interimElementProvider) {
         parent: angular.element(opts.parent),
         selectEl: element.find('md-select-menu'),
         contentEl: element.find('md-content'),
-        backdrop: opts.hasBackdrop && angular.element('<md-backdrop class="md-select-backdrop md-click-catcher">')
+        backdrop: opts.hasBackdrop && $compile('<md-backdrop class="md-select-backdrop md-click-catcher">')(scope)
```

Comment:
Same question - also, after seeing this string 3 times, I wonder if this should be shared somewhere rather than duplicated 3 times in the code.